package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@demoflow/nested-list",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
I think we're at 1.0.2 now with my change. You might want to bump this up to match to prevent issue.